avocado/core/test.py

```diff
@@ -230,6 +230,8 @@ def record_and_warn(*args, **kwargs):
         self.text_output = None
 
         self.whiteboard = ''
+        # Avoid sharing mutable default params
+        self.default_params = self.default_params.copy()
```

Comment:
True, I thought we addressed this before but I forgot... I'll remove this patch in the next version.